weave/weave_client.py

```diff
@@ -625,6 +644,78 @@ def set_call_display_name(
     def remove_call_display_name(self, call: Call) -> None:
         self.set_call_display_name(call, None)
 
+    def feedback(
+        self,
+        query: Optional[Union[Query, str]] = None,
+        reaction: Optional[str] = None,
```

Comment:
i would consider putting `reaction` as a `kw-only` arg since it is syntactic sugar over query